x/leverage/keeper/iter.go

```diff
@@ -177,12 +178,12 @@ func (k Keeper) SweepBadDebts(ctx sdk.Context) error {
 
 // GetAllUTokenSupply returns total supply of all uToken denoms.
 func (k Keeper) GetAllUTokenSupply(ctx sdk.Context) sdk.Coins {
-	return store.SumCoins(k.prefixStore(ctx, types.KeyPrefixUtokenSupply), store.NoLastByte)
+	return store.SumCoins(k.prefixStore(ctx, types.KeyPrefixUtokenSupply), keys.NoLastByte)
 }
 
 // GetAllReserves returns all reserves.
 func (k Keeper) GetAllReserves(ctx sdk.Context) sdk.Coins {
-	return store.SumCoins(k.prefixStore(ctx, types.KeyPrefixReserveAmount), store.NoLastByte)
+	return store.SumCoins(k.prefixStore(ctx, types.KeyPrefixReserveAmount), keys.NoLastByte)
```

Comment:
Doesn't this break mainnet? Leverage already stores these coins with null terminator

Reply:
ok